distros/spring/pom.xml

```diff
@@ -73,11 +81,6 @@
             <artifactId>guava</artifactId>
             <version>28.0-jre</version>
```

Comment:
❗️ These versions should not be defined in here. Versions and default scope should be in the dependency management section of the parent pom

Reply:
This seems outside of the scope of this PR. There could be other dependencies in this repo that do this as well at the moment. As long as we are not adding the versions defined here, I think a separate ticket and PR should be created to handle these issues.

Reply:
The point is that as you go through a pom file, fix it. I don't think this should be done in a separate ticket as it can easily be handled as you go.

Reply:
fixed